dali/test/python/test_operator_python_function.py

```diff
@@ -238,7 +238,7 @@ def define_graph(self):
 class BrightnessPipeline(CommonPipeline):
     def __init__(self, batch_size, num_threads, device_id, seed, image_dir):
         super(BrightnessPipeline, self).__init__(batch_size, num_threads, device_id, seed, image_dir)
-        self.brightness=ops.BrightnessContrast(device = "gpu", brightness_delta = 0)
+        self.brightness=ops.BrightnessContrast(device = "gpu", brightness = 0.5)
```

Comment:
missing tests for the other arguments

Reply:
I'll add proper tests in a separate PR..?